webapp/channels/src/components/admin_console/revoke_token_button/revoke_token_button.tsx

```diff
@@ -16,30 +16,42 @@ interface RevokeTokenButtonProps {
     onError: (errorMessage: string) => void;
 }
 
-export default class RevokeTokenButton extends React.PureComponent<RevokeTokenButtonProps> {
-    private handleClick = async (e: React.MouseEvent) => {
+const RevokeTokenButton: React.FC<RevokeTokenButtonProps> = (props) => {
+    const [error, setError] = useState<string | null>(null);
+
+    useEffect(() => {
+        if (error) {
+            props.onError(error);
+        }
+    }, [error, props]);
+
+    const handleClick = async (e: React.MouseEvent) => {
         e.preventDefault();
 
-        const response = await this.props.actions.revokeUserAccessToken(this.props.tokenId);
-        trackEvent('system_console', 'revoke_user_access_token');
+        try {
```

Comment:
Why the try/catch?

Reply:
I use it to attempt to revoke the user's token initially. However, I've come to realize that both the 'if' statement within the 'try' block and the 'catch' statement written outside serve the same purpose. Therefore, I will remove try/catch block.